v5/pivotal/projects.go

```diff
@@ -116,3 +116,39 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
@dav Ok, seems to me that I put everything under `client.Stories` (e.g. `client.Stories.ListTasks`) simply because PT is so focused on stories and it basically read the most naturally from the human perspective. But I admit that from the logical perspective it doesn't make much sense. I would still ask you to move the logic into `client.Stories.Create` and I will think about refactoring the API to add `client.Tasks` instead of having everything under `client.Stories`. But I would welcome someone else's opinion as well, e.g. yours :-)